### PR TITLE
Feat: Fix failed build log truncation

### PR DESCRIPTION
**Pull request by Meldiron**
Ensure logs of successful and failed build is not truncated.

- [x] Manual QA

Before:

CleanShot 2024-08-07 at 11 05 15@2x

After:
CleanShot 2024-08-13 at 21 34 46@2x

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -25,6 +25,7 @@ services:
     volumes:
       - /var/run/docker.sock:/var/run/docker.sock
       - ./app:/usr/local/app:rw
+      - ./vendor/utopia-php/orchestration:/usr/local/vendor/utopia-php/orchestration:rw
```
Comment (christyjacob4) [inline docker-compose.yml:28]: this needs to be removed

File: phpunit.xml [hunk 2]
```diff
@@ -6,7 +6,7 @@
          convertNoticesToExceptions="true"
          convertWarningsToExceptions="true"
          processIsolation="false"
-         stopOnFailure="false"
+         stopOnFailure="true"
```
Comment (christyjacob4) [inline phpunit.xml:9]: lets revert this

File: composer.json [hunk 3]
```diff
@@ -35,7 +35,7 @@
         "utopia-php/registry": "0.5.*",
         "utopia-php/preloader": "0.2.*",
         "utopia-php/system": "0.8.*",
-        "utopia-php/orchestration": "0.13.*",
+        "utopia-php/orchestration": "dev-fix-dockerapi-stream as 0.13.99",
```
Comment (Meldiron) [inline composer.json:38]: Release

File: composer.json [hunk 4]
```diff
@@ -35,7 +35,7 @@
         "utopia-php/registry": "0.5.*",
         "utopia-php/preloader": "0.2.*",
         "utopia-php/system": "0.8.*",
-        "utopia-php/orchestration": "0.13.*",
+        "utopia-php/orchestration": "dev-fix-curl-error as 0.13.99",
```
Comment (christyjacob4) [inline composer.json:38]: lets update the branch here

File: app/http.php [hunk 5]
```diff
@@ -646,6 +648,8 @@ function removeAllRuntimes(Table $activeRuntimes, Orchestration $orchestration):
 
             $activeRuntimes->del($runtimeName);
 
+            $message = \mb_substr($message, -1 * 1024 * 1024); // Limit to 1MB
```
Comment (christyjacob4) [inline app/http.php:651]: shouldn't this be -1 * 1000 * 1000 ?

Comment (Meldiron) [reply app/http.php:651]: That would be 1 megabit, not 1 megabyte, as per my understanding. Its typical to use bytes when talking about sizes. bits are usually only used when talking about transfer speed.

Comment (Meldiron) [reply app/http.php:651]: Setting to 1000

**Review by christyjacob4 — APPROVED** [review]

**Pull request merged**